client/reader/full-post/index.jsx

```diff
@@ -50,7 +50,9 @@ var abtest = require( 'lib/abtest' ).abtest,
 	readerRoute = require( 'reader/route' ),
 	showReaderFullPost = require( 'state/ui/reader/fullpost/actions' ).showReaderFullPost,
 	smartSetState = require( 'lib/react-smart-set-state' ),
-	scrollTo = require( 'lib/scroll-to' );
+	scrollTo = require( 'lib/scroll-to' ),
+	AuthorLink = require( 'reader/author-link' ),
+	SiteLink = require( 'reader/site-link' );
```

Comment:
That's a fair point - I suspected it might be used elsewhere but it's only being used in the full post view at the moment.